usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/README.md

```diff
@@ -0,0 +1,6 @@
+## Usecase Design
+
+This sample demonstrates cordova project app can set xwalk host, include:
+
+* '<access origin="*" />' in config of cordova project is available for setting xwalk host
```

Comment:
<access origin="*" /> is default configuation in cordova usecase. I just check it without the tag, but it is also pass.